swagger.yaml

```diff
@@ -4040,6 +4040,7 @@ definitions:
       - 'dns-domain-defined'
       - 'pull-secret-set'
       - 'ntp-server-configured'
+      - 'sufficient-hosts-count'
```

Comment:
If this is specific to single-node, then we need to improve the name
maybe 'eligible-for-standalone-mode' (or replace standalone with single-node/single-instance/non-ha)

Reply:
just a thought, why not merge the conditions and add conditions to them?
So that ``sufficient-masters-count`` can operate in HA and on standalone

Reply:
thought so too. Will change